ftplugin/markdown.vim

```diff
@@ -105,8 +105,17 @@ endfunction
 function! s:GetHeadingLinkGFM(headingName)
     let l:headingLink = tolower(a:headingName)
 
+    " \%^ : beginning of file
```

Comment:
@mzlogin Actually, I have always been in trouble with Github, when several of my PR modifies the same lines, I need to include the non merged changes I made otherwise this makes a conflict with the PR to merge. This is why I had to include previous commits which were unrelated to this PR. If you have a solution to this issue, let me know :)